Packages.csv

```diff
@@ -1506,6 +1506,7 @@ KERNEL,Kernel,XU,3.05,FAILED ACCESS ATTEMPTS LOG,HOLIDAY,10
 ,,,8992.2,ALERT RECIPIENT TYPE,,
 ,,,8992.3,ALERT CRITICAL TEXT,,
 ,,,8994.5,REMOTE APPLICATION,,
+KOREA SPECIFIC MODIFICATIONS,Korea Specific Modifications,UKO,,,,
```

Comment:
Wouldn't this be better categorized as i18n changes?

Reply:
UKO files are Korea specific. So the category is correct. I didn't think far enough of weather our changes should be part of Fileman or part of each locale's implementation, which is what you are getting at. I don't think George T will take in changes for each country.

Reply:
I'm OK with that.